### PR TITLE
fix for new LiveScript package name (livescript)

### PR DESCRIPTION
**Pull request by vendethiel**
Since LS1.4.0 went out, the package is now "livescript". You'll probably need to rename "LiveScript-brunch" as well.

Added the `header: no` as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "LiveScript-brunch",
-  "version": "1.6.0",
+  "name": "livescript-brunch",
+  "version": "1.7.0",
```
Comment (es128) [inline package.json]: please don't bump version in a PR

File: src/index.coffee [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = class LiveScriptCompiler
 
   compile: (data, path, callback) ->
     try
-      result = LiveScript.compile data, bare: yes
+      result = livescript.compile data, bare: yes, header: no
```
Comment (es128) [inline src/index.coffee]: Could you please explain the impact of `header: no`?

Comment (vendethiel) [inline src/index.coffee]: It removes the "// Generated by LiveScript version 1.4.0" header

File: package.json (hunk 1 above)
Comment (vendethiel) [inline package.json]: sorry, want me to commit this out?

File: src/index.coffee (hunk 2 above)
Comment (es128) [inline src/index.coffee]: What if users want that? Perhaps it should be an option that gets exposed to the brunch config if this is something you want to use.

File: package.json (hunk 1 above)
Comment (es128) [inline package.json]: yes please

File: src/index.coffee (hunk 2 above)
Comment (vendethiel) [inline src/index.coffee]: that seems to be the scope of another PR, though. This message didn't exist "back then"
(and having the header prevents the test from passing)

Comment (es128) [inline src/index.coffee]: Ok fair enough. Didn't realize the header was something new on livescript's side.

**Comment by vendethiel** [comment]
Removed the version number change!

**Comment by es128** [comment]
Thanks!

**Comment by vendethiel** [comment]
Thanks! I'll wait for the version number update/npm tag.

**Comment by es128** [comment]
https://www.npmjs.com/package/livescript-brunch

**Comment by vendethiel** [comment]
thanks :-)
